### ISSUE TITLE
Connecting Diagrams with Tables

### ISSUE THREAD

**Issue by PrzemyslawKlys** [EvotecIT/PSWriteHTML#63]
"I was wondering if there was some functionality that would allow you to filter/fold/hide DiagramNodes and DiagramLinks interactively from the web page?"

Answer: "You can see here different examples what the library can do: https://visjs.github.io/vis-network/examples/

It would take some time to make that available in PowerShell and make that easy to use, but should be possible.

I believe you want something similar to that: https://visjs.github.io/vis-network/examples/network/other/clusteringByZoom.html?  This works folding/unfolding when zooming.

I was also experimenting by connecting click event with table so when node is clicked a table row with same name would highlight. But again this require some work on PS+JS side. "

**Comment by thomwatt** [comment]
Yes, I would really like to see this feature also. I use your module to produce infrastructure diagrams that can get very busy. The ability to collapse and expand nodes would make the diagram navigation much easier and a more valuable artefact for the reader. I've tried the zoom feature but it doesn't feel like it would be as useful as the ability to choose the node to expand/collapse.

Loving your work by the way - if the above is possible and with the new linked table feature, your awesome module would become double awesome!

Tom
